src/app/shell/admin-tools/data/history-log/history-log.component.ts

```diff
@@ -1,28 +1,32 @@
+import { Observable, Subject } from 'rxjs';
+import {
+    debounceTime, distinctUntilChanged, filter, map, startWith, takeUntil
```

Comment:
Looks like there are 4 spaces instead of 2.
If you use an extension to sort imports, add this to the settings config file
`"importSorter.importStringConfiguration.tabSize": 2`